### DISCUSSION TITLE
maps with note specific ar, cs and od

### DISCUSSION THREAD

**Discussion by oliverlundstrom** [ppy/osu#29220] [Ideas] [answered]
Would it be possible to somehow be able to make maps that have a modifier for each circle and slider placed, you could click the circle in the editor and change that specific circles ar, cs or od.
This would make mapping infinitely more custom.
Sorry if this has already been talked to death, I tried to search for this for a while on here but couldnt find anything and this has been an idea i've had for a while.

**Comment by oliverlundstrom** [comment]
just imagine aiming breaks where you just need to tap because the circle is the whole screen. Imagine patterns where circles change size. Imagine how interactive and expressive mapping could be if ar could react like notes do in songs, with velocity?

**Comment by oliverlundstrom** [comment]
this is probably what osu 2 would be

**Comment by peppy** [comment, marked as answer]
Please stop spamming github.

Use search. Basically everything you've posted already has threads or answers.

**Reply by oliverlundstrom** [reply]
I tried for multiple hours, I do apologize like I have been. Ill just make a youtube video with my ideas that will be a better way to show them if you are interested at all.

**Reply by oliverlundstrom** [reply]
I am trying to help so yeah. Sorry for being a little mentally ill, I will stop posting here.

**Reply by oliverlundstrom** [reply]
sorry to ask but if you could please link me a thing where this is discussed if its so easy to find, I would appriciate it. I did search for a while.

**Reply by Joehuu** [reply]
https://github.com/ppy/osu/discussions/15577

**Reply by oliverlundstrom** [reply]
wide range of ideas are important while planning for the future. I am sorry though and wont post again. If you see this, do let me know if I should even bother putting this passion into ideas and a video for this game I love or not, even though this might sound sarcastic I really dont want to annoy you or get in your way, whoever it is thats reading this, so let me know directly and I'll act as you wish.

**Reply by oliverlundstrom** [reply]
one more thing then ill stop. When I learn coding properly and have worked for a while and understand the culture, I will be back to help yall. (if you want.) Thank you for making this game. I really appriciate you as people a lot.
